requirements/doc.txt

```diff
@@ -23,7 +23,7 @@ asynctest==0.13.0
     # via
     #   -r requirements/test.txt
     #   aiohttp
-attrs==21.4.0
+attrs==22.1.0
```

Comment:
Only major change is the drop of support for Python2.7
Change-logs: https://www.attrs.org/en/stable/changelog.html#backwards-incompatible-changes